components/engine_traits/src/options.rs

```diff
@@ -147,8 +171,8 @@ impl Default for IterOptions {
             lower_bound: None,
             upper_bound: None,
             prefix_same_as_start: false,
-            fill_cache: false,
-            key_only: false,
+            fill_cache: true,
+            titan_key_only: false,
```

Comment:
I think `key_only` does not equal to `titan_key_only`, when we don't use titan we also can use key_only, for example select count(*) doesn't read values but only read committed information in write cf.

Reply:
@zhangjinpeng1987 As you said, `select count(*) doesn't read values` is controlled by `omit_value` on scanner level. But here `key_only` is actually for engine level. Cause `Rocksdb` doesn't support key_only and has no related options, the `key_only` is now for titan specifically.

Reply:
I think we can keep the name `key_only` for other engines